### PR TITLE
Remove the reference to "jwt_disable_XYZ_validation" which are no lon…

### PR DESCRIPTION
**Pull request by letzya**
…ger in use

### CODE REVIEW

File: tyk-docs/content/security/your-apis/json-web-tokens.md [hunk 1]
```diff
@@ -116,35 +116,24 @@ Instead of specifying static public key in API definition, it is possible to spe
...
-You can now configure JWT clock skew using the following variables. All values are in seconds. The default is `0`.
+You can now configure JWT clock skew using the following variables. All values are in seconds. The default is `0` (i.e. no skew).
 
 ```{.json}
 "jwt_issued_at_validation_skew": 0,
 "jwt_expires_at_validation_skew": 0,
 "jwt_not_before_validation_skew": 0
 ```
 
+The fields to disable the validation of the 3 claims `jwt_disable_XYZ_at_validation` are no longer in use since the new ones gives more flexability
```
Comment (buger) [inline tyk-docs/content/security/your-apis/json-web-tokens.md]: I this we can remove this field at all. It was used only by one customer.

Comment (marksou) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: So we can remove the reference to `jwt_disable_XYZ_at_validation`?

Comment (buger) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: Yep

**Review by marksou — APPROVED** [review]
Removed `jwt_disable_XYZ_at_validation` reference as per @buger comment.

File: tyk-docs/content/security/your-apis/json-web-tokens.md (hunk 1 above)
Comment (letzya) [reply tyk-docs/content/security/your-apis/json-web-tokens.md]: even better, thanks for checking

**Pull request merged**